account_payment_mode/models/account_payment_mode.py

```diff
@@ -68,6 +68,7 @@ class AccountPaymentMode(models.Model):
     )
     active = fields.Boolean(default=True)
     note = fields.Text(translate=True)
+    sequence = fields.Integer()
```

Comment:
It's the usual default. Not sure if 0 can cause any trouble.

```suggestion
    sequence = fields.Integer(default=10)
```